README.md

```diff
@@ -53,6 +53,8 @@ generic with minimal need for project-specific behavior.
 
 - AWS credentials configured in `~/.aws/credentials` and/or `~/.aws/config`
 
+- macOS Big Sur (11.x) users should follow additional steps mentioned in [Troubleshooting](#5-troubleshooting).
```

Comment:
This can be specified even further to the correct heading that was added to the Troubleshooting section.
ex.
https://github.com/DataBiosphere/azul/blob/f77c031973c7ffe3d30dd4f5477fb800850c55e0/README.md#macos-big-sur-pyenv-build-failed